lib/api/src/rt/wamr/utils/convert.rs

```diff
@@ -79,10 +86,7 @@ impl IntoWasmerValue for wasm_val_t {
             bindings::wasm_valkind_enum_WASM_EXTERNREF => {
                 panic!("ExternRefs are not currently supported through wasm_c_api")
             }
-
-            _ => {
-                panic!("wamr currently does not support V128 values")
-            }
+            _ => unreachable!("wamr kind has no matching wasmer_types::Type"),
```

Comment:
This is not the right type. We should format well

Reply:
[b3be2ef](https://github.com/wasmerio/wasmer/pull/5347/commits/b3be2efc39fe2168f75fba91941374241ef5480d)